modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/query/AbstractADeleteQuery.java

```diff
@@ -90,7 +90,7 @@ public AbstractADeleteQuery(AssetManagerImpl am, String owner) {
     };
   }
 
-  public long run(DeleteSnapshotHandler deleteSnapshotHandler) {
+  public long run(DeleteEpisodeHandler deleteSnapshotHandler) {
```

Comment:
```suggestion
  public long run(DeleteEpisodeHandler deleteEpisodeHandler) {
```

The one thing that is still called `deleteSnapshotHandler` or similar.

Reply:
Fixed